_pages/dev/connecting-to-other-systems.md

```diff
@@ -1,74 +1,70 @@
 ---
-title: Connecting to Other Systems (DRAFT)
+title: Connecting to Other Systems
 ---
 
-The system landscape of a Commerce solution is made up of various systems. The systems are typically orchestrated on various application layers, including the front end. While the Spartacus component UIs connect to SAP Commerce Cloud APIs by default, the underlying framework can be used to _adapt_ other systems as well. This is done by so-called connectors which can be added to connect to other systems.
+The system landscape of a Commerce Cloud solution is made up of various systems, which are typically orchestrated on various application layers, including the front end. The Spartacus component UIs connect to SAP Commerce Cloud APIs by default, but the underlying framework can be used to work with other systems as well. This is done by so-called "connectors", which can be added to connect to other systems.
 
 ## Component Data Binding
 
-Spartacus delivers view logic that binds to (complex) commerce data and logic in the commerce back end. Angular provides standards for data binding and relies on reactive programming as the best-practice standard pattern for data binding. A number of best practices are used for data binding:
+Spartacus delivers view logic that binds to (complex) commerce data and logic in the Commerce Cloud back end. Angular provides standards for data binding, and relies on reactive programming as the best-practice, standard pattern for data binding. The following best practices are used for data binding in Spartacus:
 
-- UI components bind to observable data from the back end, using Angular's standard `async`.
-- UI components do not store response data from observables locally, so that destroy logic can be avoided. RxJS _pipeable_ logic can be applied to implement any logic when data is observed.
-- back end data is stored in a central data store, provided by a state management system (Spartacus uses ngrx).
-- The complexity of the state management system is fronted by a facade layer to provide a simple API to component developers.
-- The back end system is configurable by connector, adapter and convertor logic. Customers can provide alternative implementations to adapt a specific back end.
+- UI components bind to observable data from the back end, using the standard Angular `async`.
+- UI components do not store response data from observables locally, which means destroy logic can be avoided. RxJS pipeable logic can be applied to implement any logic when data is observed.
+- Back-end data is stored in a central data store, provided by a state management system. Spartacus uses NgRx.
```

Comment:
I see sometimes _back end_ and sometimes _back-end_. Is there an explicit difference?

Reply:
Yes. I started out wondering if I should just make everything `backend`, though I had been following the SAP style guide to make it `back end`... So I did further research and found there is general agreement in the world that `back end` is the noun (communicates with the back end), and that `back-end` is the adjective (back-end data, back-end model, back-end system, etc). So I will be making it like this throughout the docs from now on...

Reply:
Nice, I use https://books.google.com/ngrams/graph?content=backend%2Cback+end%2C+back-end&year_start=1800&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cbackend%3B%2Cc0%3B.t1%3B%2Cback%20end%3B%2Cc0%3B.t1%3B%2Cback%20-%20end%3B%2Cc0 often when I'm unsure. It's just a comparison of what's used often, not necessarily the right thing though.